setup.py

```diff
@@ -24,9 +25,11 @@
 if platform.system() == "Windows":
     libraries.extend(["libprotobuf-lite", "valhalla", "libcurl", "zlib", "Ws2_32", "ole32", "Shell32"])
     extra_compile_args.extend(["-DNOMINMAX", "-DWIN32_LEAN_AND_MEAN", "-DNOGDI"])
+    extra_objects = []
 else:
-    libraries.extend(["protobuf-lite", "valhalla", "curl", "z"])
+    libraries.extend(["/usr/local/lib/libprotobuf-lite.a", "valhalla", "curl", "z"])
     extra_link_args.extend(["-lvalhalla", "-lprotobuf-lite", "-lcurl", "-lz"])
+    extra_objects = ["/usr/local/lib/libprotobuf-lite.a"]
```

Comment:
actually it doesn't make sense to use the library from the system path, we should also copy both the .a and include dir (that we already do, but still reference the system path, just remove that)